tests/converters/app-converter.test.ts

```diff
@@ -191,64 +191,236 @@ describe('App Converter', () => {
     });
 
     test.each([
-        ['ref', createStackManifest({ Bucket: { Ref: 'resource1' } }), 'resource1_id'],
+        ['ref', createStackManifest({ Bucket: { Ref: 'resource1' } }), 'resource1_id', undefined],
```

Comment:
Would it be possible to have the undefined labeled somehow? A record perhaps where things are omitted instead of having to spell undefined? Gets a bit hard to read.